docs/share_plus/usage.mdx

```diff
@@ -29,14 +29,14 @@ sharing to email.
 Share.share('check out my website https://example.com', subject: 'Look what I made!');
 ```
 
-To share one or multiple files invoke the static `shareFiles` method anywhere in your Dart code. Optionally you can also pass in `text` and `subject`.
+To share one or multiple files invoke the static `shareXFiles` method anywhere in your Dart code. Optionally you can also pass in `text` and `subject`.
 
 ```dart
-Share.shareFiles(['${directory.path}/image.jpg'], text: 'Great picture');
-Share.shareFiles(['${directory.path}/image1.jpg', '${directory.path}/image2.jpg']);
+Share.shareXFiles(['${directory.path}/image.jpg'], text: 'Great picture');
+Share.shareXFiles(['${directory.path}/image1.jpg', '${directory.path}/image2.jpg']);
 ```
 
-If you are interested in the action your user performed with the share sheet, you can instead use the `shareWithResult` and `shareFilesWithResult` methods.
+If you are interested in the action your user performed with the share sheet, you can instead use the `shareWithResult` and `shareXFiles` methods.
```

Comment:
This sentence maybe doesn't make sense anymore.

Before the deprecation:
`shareFiles` returns void.
`shareWithResult` and `shareFilesWithResult` return `ShareResult`.

Now:
`shareXFiles` also returns a `ShareResult`.

Reply:
@nohli, I've re-write the minimal code please review it.

To share one or multiple files invoke the static `shareXFiles` method anywhere in your Dart code and it'll return the `ShareResult`. Optionally you can also pass in `text` and `subject`.

```dart
final result = await Share.shareXFiles(['${directory.path}/image.jpg'], text: 'Great picture');

if (result.status == ShareResultStatus.dismissed) {
    print('did you not like the picture with text?');
}
```

```dart
final result = await  Share.shareXFiles(['${directory.path}/image1.jpg', '${directory.path}/image2.jpg']);

if (result.status == ShareResultStatus.dismissed) {
    print('did you not like the pictures?');
}
```

If you are interested in the action your user performed with the share sheet, you can instead use the `shareWithResult` method.

```dart
final result = await Share.shareWithResult('check out my website https://example.com');

if (result.status == ShareResultStatus.success) {
    print('thank you for sharing my website!');
}
```

Reply:
I think the part `If you are interested in the action your user performed with the share sheet` is still wrong.

It implies that the former method doesn't return a result - which was true for `shareFiles`, and that's why it was written. After changing to `shareXFiles`, which returns a result, the sentence doesn't make sense at this point.